examples/encfs/README.md

```diff
@@ -109,9 +109,9 @@ lost+found  test.txt
 [!] Closing device...
 ```
 
-The user needs to upload the blob to the previously generated storage container by uploading to the Azure Portal, using [az storage blob upload] (https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli), or using [azcopy] (https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json).
+The user needs to upload the blob to the previously generated storage container by uploading to the Azure Portal, using [az storage blob upload] (https://learn.microsoft.com/en-us/azure/storage/blobs/storage-quickstart-blobs-cli), or using [azcopy] (https://learn.microsoft.com/en-us/azure/storage/common/storage-use-azcopy-v10?toc=%2Fazure%2Fstorage%2Fblobs%2Ftoc.json&bc=%2Fazure%2Fstorage%2Fblobs%2Fbreadcrumb%2Ftoc.json). When uploading the blob, the type must be specified as a "page blob" for a read-write filesystem.
 
-```az storage blob upload --file generatefs/encfs.img --container-name <storage container name> --name <name of block blob of the image generated> --account-name <storage account name> --sas-token '<SAS token starting with sp within single quotations>'```
+```az storage blob upload --file generatefs/encfs.img --container-name <storage container name> --name <name of blob of the image generated> --account-name <storage account name> --type <page or block blob> --auth-mode login```

```

Comment:
Is the sas token functionality removed or just not recommended anymore?

Reply:
There's a lot of ways to authenticate. Personally, I prefer the auth-mode login (means you don't have to do anything if you're logged into an account with the appropriate access) and think it might be easier for customers to use (no extra step of generating a token). However, they can use the SAS token or even a connection string. There is a link to the documentation on az storage blob and if the customers wish to use a different mode of authentication they can look there.